src/main/scala/dpla/ingestion3/mappers/providers/MoMapping.scala

```diff
@@ -85,6 +92,9 @@ class MoMapping extends JsonMapping with JsonExtractor with IdMinter[JValue] wit
   override def title(data: Document[JValue]): AtLeastOne[String] =
     extractStrings(unwrap(data) \ "sourceResource" \ "title")
 
+  override def `type`(data: Document[JValue]): AtLeastOne[String] =
+    extractStrings(unwrap(data) \ "sourceResource" \ "type")
+
```

Comment:
In the `Mapping` trait, the `type` method has a different return type:
```
def `type`(data: Document[T]): ZeroToMany[String] = Seq()
```
Do you want the return types to agree?

Reply:
doh -- yes